### PR TITLE
Add OLS 2020 proposal

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: proposals/openlifescience_2020.Rmd [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+max. 200 words  
+How and which parts of pre-existing documentation to improve that will provide the most value for the effort. Also, how to make it easier and incentivize both our group and potential users to improve documentation for our datasets and code. 
+
+**What do you expect from a mentor?**  
+max. 200 words  
+We are looking for a mentor who has experience in developing communities around shared software tools, as well as with writing the documentation and other materials that enable these communities. We would also appreciate a mentor who can help us to assess the effectiveness of the outreach strategies, such as documentation and training, that we create. 
+
+**Do you speak any languages other than English?**  
+If yes, please list them.  
+No.
```
Comment (dlebauer) [inline proposals/openlifescience_2020.Rmd]: @Sithyphus and @MagicMilly is this correct?

**Comment by KristinaRiemer** [comment]
@dlebauer I've been including mds for things that people who might not be familiar with Rmarkdown/markdown in general might possibly look at, such as vignettes, because they're rendered on GitHub and are easy to look at. Maybe there's a better way to do that?

**Comment by MagicMilly** [comment]
That is correct for me - English only.

**Comment by Sithyphus** [comment]
I speak Spanish, trying to learn Japanese in case it ever comes up.

**Pull request merged**